src/Shared/LayoutRenderers/AspNetRequestHttpMethodLayoutRenderer.cs

```diff
@@ -17,7 +17,7 @@ namespace NLog.Web.LayoutRenderers
     /// ${aspnet-request-method}
     /// </remarks>
     [LayoutRenderer("aspnet-request-method")]
-    public class AspNetRequestHttpMethodRenderer : AspNetLayoutRendererBase
+    public class AspNetRequestHttpMethodLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
Though it seems tempting, then you can repair past mistakes with major-version-bump (Ex. NLog.Web v6)

Please revert, then we can make an issue for v6 about fixing this.